examples/views/pagination.py

```diff
@@ -0,0 +1,72 @@
+import discord
+from discord.ext import commands
+
+
+class PaginationView(discord.ui.View):
+    current = 0 # It's not in the constructor because it'll be reset everytime the view is updated.
+    def __init__(self, embed_list):
+        super().__init__()
+        self.embed_list = embed_list
+        if len(embed_list) == 1:
+            self.next.disabled = True
+            self.next.style = discord.ButtonStyle.red
+        self.count.label = f"Page {self.current + 1}/{len(self.embed_list)}"
+
+
+    # To make sure only ctx.author can interact.   
+    async def interaction_check(self, interaction: discord.Interaction) -> bool:
+        if self.user == interaction.user:
+            return True
+        await interaction.response.send_message(f'Only `{self.user.name}` can interact. Run the command if you want to.', ephemeral=True)
+        return False
+        
+    @discord.ui.button(label="Previous", style=discord.ButtonStyle.red, disabled=True) # you can add emojis too..
+    async def previous(self, button, interaction):
+        self.current -= 1
+        self.next.disabled = False
```

Comment:
Why is this here? Is this implying that `embed_list` can be modified and next should be updated to reflect that? If so, then there is no handling of to disabling the button later